pkg/provision/source_test.go

```diff
@@ -5,36 +5,72 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
 	"github.com/threefoldtech/zos/pkg"
 )
 
-type testReservationStore struct {
-	fReserve func(r Reservation, nodeID pkg.Identifier) error
-	fPoll    func(nodeID pkg.Identifier, all bool, since time.Time) ([]*Reservation, error)
-	fGet     func(id string) (Reservation, error)
+type TestPollSource struct {
```

Comment:
should this be exported ?

Reply:
It doesn't really matter because it's defined in a _test.go file so it doesn't get exported.

:smile: